community/merchants/index.md

```diff
@@ -4,74 +4,110 @@ title: titles.merchants
+      <div class="right half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+        <div class="info-block">
+          <div class="center-xs">
+            <img class="merch" src="/img/merchants/localmonero.png" alt="LocalMonero logo" title="LocalMonero">
+            <p>{% t merchants.locmondescr %}</p>
+          </div>
+          <div>
+            <p><b>{% t merchants.cardfoss %}</b> <span class="cross"></span></p>
+            <p><b>{% t merchants.noncustodial %}</b> <span class="cross"></span></p>
+            <p>{% t merchants.cardkyc %} {% t merchants.depsell %}</p>
+            <p><b>XMR &#8596; fiat:</b> <span class="check"></span></p>
+            <p><b>{% t merchants.cardonion %}</b> <span class="check"></span></p>
+          </div>
+          <div class="center-xs">
+            <p><a href="https://localmonero.co" class="btn-link btn-fixed btn-primary-top">{% t merchants.visitlocalmonero %}</a></p>
+          </div>
         </div>
+      </div>
+      <div class="left half no-pad-sm col-lg-6 col-md-6 col-sm-12 col-xs-12">
+        <div class="info-block">
+          <div class="center-xs">
+            <img class="merch" src="/img/merchants/hodlhodl.jpg" alt="HodlHodl Logo" title="HodlHodl">
+            <p>{% t merchants.hodlhodldescr %}</p>
+          </div>
+          <div>
+            <p><b>{% t merchants.cardfoss %}</b> <span class="cross"></span></p>
+            <p><b>{% t merchants.noncustodial %}</b> <span class="check"></span></p>
+            <p>{% t merchants.cardkyc %} <span class="check"></span></p>
+            <p><b>XMR &#8596; fiat:</b> <span class="cross"></span> {% t merchants.cardbtcfiat %}</p>
+            <p><b>{% t merchants.cardonion %}</b> <span class="cross"></span></p>
+          </div>
+          <div class="center-xs">
+            <p><a href="https://hodlhodl.com" class="btn-link btn-fixed btn-primary-top">{% t merchants.visithodlhodl %}</a></p>
+          </div>
         </div>
-</div>
-{%endfor%}
-
-</div>
-<div class="text-center container description">
-    <p>
-        <em>{% t merchants.disclaimer %}</em>
-    </p>
-</div>
-
-<a class="arrow-up" href="#" aria-label="{% t accessibility.arrowup %}" title="{% t accessibility.gotop %}"><i></i></a>
+      </div>
+    </div>
+    <div class="full col-lg-12 col-md-12 col-sm-12 col-xs-12">
+      <div class="info-block">
+        <h2>{% t merchants.centrexchanges %}</h2>
+        <p>{% t merchants.centrexchangesp %}</p>
+        <h3>CEXes</h3>
+        <p>{% t merchants.cexp %}</p>
+        <ul class="logo">
+            <li><a href="https://www.kraken.com/">Kraken</a> (EUR, USD, CAD, GBP, JPY, AUD, CHF)</li>
+            <li><a href="https://www.binance.com/trade.html?symbol=XMR_BTC">Binance</a> (USD, EUR, RUB, TRY, NGN, UAH, KZT, INR, ...)</li>
+            <li><a href="https://dvchain.co/">DV Chain (OTC)</a> (USD, CAD, GBP, EUR, JPY, ...)</li>
+            <li><a href="https://www.bitfinex.com/">Bitfinex</a> (USD)</li>
+          </ul>
```

Comment:
```suggestion
            <li><a href="https://www.kraken.com/">Kraken</a> (EUR*, USD*, CAD, GBP, JPY, AUD, CHF)</li>
            <li><a href="https://www.binance.com/trade.html?symbol=XMR_BTC">Binance</a> (USD, EUR, RUB, TRY, NGN, UAH, KZT, INR, ...)</li>
            <li><a href="https://dvchain.co/">DV Chain (OTC)</a> (USD*, CAD*, GBP*, EUR*, JPY*, ...)</li>
            <li><a href="https://www.bitfinex.com/">Bitfinex</a> (USD*)</li>
          </ul>
          <p>*Fiat currency to Monero trading pair (e.g. USD/XMR, EUR/XMR)</p>
```

Reply:
What if we add which of these fiat currencies have direct trading pair with Monero?

Reply:
added